packages/eslint-plugin-ckeditor5-rules/lib/rules/no-relative-imports.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+module.exports = {
+	meta: {
+		type: 'problem',
+
+		docs: {
+			description: 'Disallow relative imports from CKEditor5 packages',
+			category: 'CKEditor5'
+		},
+		fixable: 'code',
+		schema: [] // no options
```

Comment:
```suggestion
		schema: []
```

This comment is so obvious and can be removed. We can see without the comment that there are no options. Or if it should stay, let's clarify why there are no options.